src/modules/ChannelSettings/components/ChannelProfile/index.tsx

```diff
@@ -28,10 +29,10 @@ const ChannelProfile: React.FC = () => {
   const channel = channelSettingStore?.channel;
 
   const channelName = useMemo(() => {
-    if (channel?.name && channel.name !== 'Group Channel') {
+    if (!isDefaultChannelName(channel)) {
       return channel.name;
     }
-    if (channel?.name === 'Group Channel' || !channel?.name) {
+    if (isDefaultChannelName(channel)) {
```

Comment:
It can be just 
```
if (isDefaultChannelName(channel)} { return ... }
else { return channel.name }
```

Reply:
I Applied it. And additionally, I aligned the logic for the `stringSet.NO_TITLE` like below.
```
if (!channel?.name && !channel?.members) return stringSet.NO_TITLE;
```